pkg/management/component.go

```diff
@@ -490,9 +490,14 @@ func (c *component) ExecuteActionsEmail(ctx context.Context, realmName string, u
 	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
 
 	var actions = []string{}
+	var init_password_action = "sms-password-set"
```

Comment:
You might have a go-lint recommandation asking not to use underscore in variable names. initPasswordAction would be more Go-compliant

Reply:
Hahaha, only now I noticed how I named this variable (Friday tiredness) :))). 
Thanks, good point.